### ISSUE TITLE
[a11y audit] Forms

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#4920]
* Fields that have required fields should be read as such. If they aren't, they will need an `aria-required="true"` on them, which would be good to have on all required fields regardless.
* Fields that trigger an error should be read the error. If it's not clear from a screenreader that a field has turned invalid, they should receive `aria-invalid="true"`.
* On error, there should be 1) a clear visual indicator (like a red border) on errored fields, 2) an error message read stating what went wrong (even text that says "please fill out required fields" is sufficient, or a tooltip) that's accessible, and 3) keyboard focus should be directed back to the first invalid field.
* Ensure labels are read after "animate floating labels" task is complete.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1623)
┆Issue Number: FXA-1623

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Closing as we will request auditing of new Settings from our partners in a11y.
